geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/io/geotiff/GeoTiffRDD.scala

```diff
@@ -79,35 +81,57 @@ object GeoTiffRDD {
     }
   }
 
-  def get(
+  def get_many(
```

Comment:
Also I think its safe only to have the version that requires options, which could be empty. Am I missing anything ?